core/configuredivecomputerthreads.cpp

```diff
@@ -883,28 +921,19 @@ static dc_status_t write_ostc4_settings(dc_device_t *device, const DeviceDetails
 		return rc;
 	EMIT_PROGRESS();
 
-
-	//sync date and time
-	if (deviceDetails.syncTime) {
-		dc_datetime_t now;
-		dc_datetime_localtime(&now, dc_datetime_now());
-
-		rc = dc_device_timesync(device, &now);
-	}
-
+	if (rc == DC_STATUS_SUCCESS)
```

Comment:
This check is redundant. If rc is anything but DC_STATUS_SUCCESS , we would already have returned.,